tbtc-ts/src/bitcoin.ts

```diff
@@ -17,6 +17,23 @@ export interface Transaction {
    */
   transactionHash: string
 
+  /**
+   * The block hash of the transaction's containing block as an un-prefixed
+   * hex string.
+   */
+  blockHash: string
+
+  /**
+   * The number of confirmations the transaction has received, including the
+   * containing block hash.
+   */
+  confirmations: number
+
+  /**
+   * The full transaction payload as an un-prefixed hex string.
+   */
+  hex: string
```

Comment:
If we decide to follow the approach proposed in https://github.com/keep-network/tbtc-v2/pull/86#discussion_r819531383 we won't need those fields.

Reply:
Done.